test/rackup/hello-logs.ru

```diff
@@ -0,0 +1,15 @@
+require 'logger'
+class AppWithLogger
+  def initialize
+    @logger = Logger.new('test/log.log')
+  end
+
+  def call(env)
+    puts "write to log"
+    @logger.info "hello"
+    #p @logger.instance_variable_get(:@logdev)
```

Comment:
Is this debugging which you want to keep in the test file?

Reply:
removed